lib/error.js

```diff
@@ -46,27 +66,21 @@ const serializeMessageItem = messageItem => {
 export const makeError = ({
 	stdio,
 	all,
-	error,
-	signal,
-	exitCode,
+	error: rawError,
+	signal: rawSignal,
+	exitCode: rawExitCode,
 	command,
 	escapedCommand,
 	timedOut,
 	isCanceled,
-	options: {timeoutDuration: timeout, cwd},
+	options: {timeoutDuration, timeout = timeoutDuration, cwd},
```

Comment:
Fixes a bug where the error message of timeouts with `execaSync()` would show `Command failed with undefined milliseconds`.